### PR TITLE
[Events backend part 2] Added controller for event model. Add root route #index

### PR DESCRIPTION
**Pull request by shamritskiy3468**
https://trello.com/c/ttfwdMnO

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Events controller class
+class EventsController < ApplicationController
+
+  before_action :set_event, only: [:show, :edit, :update, :detroy ]
+
+  def index
+    @events = Event.all
+  end
+
+  def new
```
Comment (nplusp) [inline app/controllers/events_controller.rb]: All creating logic will be perfromed from the admin panel. No need of having these actions in this controller.
Just an index and show action will be enough.

File: config/database.yml.example [hunk 2]
```diff
@@ -82,4 +82,4 @@ production:
   <<: *default
   database: minskrb_production
   username: minskrb
-  password: <%= ENV['MINSKRB_DATABASE_PASSWORD'] %>
```
Comment (alexshgov) [inline config/database.yml.example]: Why did you change the example ?
